### DISCUSSION TITLE
Can't echo data

### DISCUSSION THREAD

**Discussion by KindSpidey** [Cyfrin/foundry-full-course-cu#813] [Q&A]
Hey guys!

I work on Lesson 7, where we need to print our RPC_URL and PRIVATE_KEY. Can you tell me, please, why my echo command leads to empty strings?

my .env file is:

```
PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
RPC_URL=http://127.0.0.1:8545
```
no whitespaces as you see

then I do this in terminal:

```
source .env
echo $RPC_URL
```
and it prints me empty line
why and how to solve it?

**Comment by theirrationalone** [comment]
```bash
source Users/your_username/.bashrc
source .env
```

**Reply by KindSpidey** [reply]
no matter, which username I use, username of PC user or Linux profile, I got  No such file or directory

**Reply by theirrationalone** [reply]
Try these... 

```bash
source ../.bashrc
```

Then

```bash
source ./.env
```

Then

```bash
echo $RPC_URL
```

**Reply by KindSpidey** [reply]
start from which directory? I have this structure:

`\\wsl.localhost\Ubuntu\home\kindspidey\newFolder\foundry-simple-storage-f23`

and if I am in my last folder or in home folder, command `source ../.bashrc` in both cases leads to `bash: ../.bashrc: No such file or directory`

**Reply by theirrationalone** [reply]
from your lesson7 project directory..... 

For bashrc, you can skip that

**Reply by KindSpidey** [reply]
Tried, still does not work

**Reply by theirrationalone** [reply]
Now we have only one choice... 

Uninstall wsl linux. 
Install again
Install foundry again. 

Don't forget to copy your crucial work (contracts/projects)

**Reply by KindSpidey** [reply]
Oh my God, I found the solution, it is hilarious. I just didnt save my file, so because of this it didnt see any variables in it. Anyway thanks for helping
